### PR TITLE
Extract welcome messages to dictionary

### PR DESCRIPTION
**Pull request by mslowiak**
## Associated Issue

Part of #99 

## Implemented Solution

Extracted messages to dictionary class

## Testing Details

No tests needed

### CODE REVIEW

File: src/main/java/com/hydratereminder/HydrateReminderPlugin.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 package com.hydratereminder;
 
 import com.google.inject.Provides;
+import com.hydratereminder.dictionary.HydrateWelcomeMessageDictionary;
```
Comment (jmakhack) [inline src/main/java/com/hydratereminder/HydrateReminderPlugin.java:29]: For the other dictionary class, the message retrieval method was directly imported into this class: 
`import static com.hydratereminder.dictionary.HydrateBreakMessageDictionary.getRandomHydrateBreakMessageForPersonality;`

Can we do the same for this one as well to keep things consistent?

**Review by jmakhack — CHANGES_REQUESTED** [review]
The changes look good and I was actually going to suggest this change after what you did in the previous pull request but you beat me to it haha :)

Looks like there is a merge conflict after I merged your other pull request. Can you take a look into resolving it?

File: src/main/java/com/hydratereminder/HydrateReminderPlugin.java (hunk 1 above)
Comment (mslowiak) [reply src/main/java/com/hydratereminder/HydrateReminderPlugin.java]: Yup, good catch!

**Comment by mslowiak** [comment]
@jmakhack fixed

**Review by jmakhack — APPROVED** [review]

**Pull request merged**